src/app.ts

```diff
@@ -1,26 +1,65 @@
-// example interfaces that can be use
-// TIP: the types mentioned in the interfaces must be fulfilled in order to solve the problem.
-interface TemperatureReading {
-  time: Date
-  temperature: number
-  city: string
-}
-interface TemperatureSummary {
-  first: number
-  last: number
-  high: number
-  low: number
-  average: number
+import { TemperatureReading, TemperatureSummary } from './interfaces'
+
+type ProccesedReadings = {
+  [city: string]: {
+    [date: string]: number[]
+  }
 }
+let processedReadings: ProccesedReadings = {}
```

Comment:
If we don't need a reassignment we need to use const instead.
ref: [see here](https://www.shecodes.io/athena/124459-why-are-we-using-let-and-not-const#:~:text=The%20main%20difference%20between%20them,variable%20may%20change%20over%20time.)